src/Arith.jl

```diff
@@ -254,7 +294,7 @@ Base.:-(x::Cyclotomic{<:NfPoly}, y::Union{Int64, Rational{Int64}}) = x-(y*one(x)
 Base.:-(y::Cyclotomic) = (-1)*y
 
 Base.:(==)(x::CycloSum, y::CycloSum) = iszero(x-y)
-Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero(x-y)
+Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero(x-y, ignore_exceptions=true)
```

Comment:
.... or just do this?
```suggestion
Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero((x-y).numerator)
```

Reply:
Yes, that's also fine.